DiscordBot/Program.cs

```diff
@@ -88,29 +86,36 @@ private static async Task MainAsync()
                 .GetMemberAsync(eventArguments.User.Id).Result
                 .CreateDmChannelAsync();
 
-            if (eventArguments.Emoji.GetDiscordName() == ":no_entry_sign:")
+            switch (eventArguments.Emoji.GetDiscordName())
             {
-                await eventsSheetsService.ClearResponsesForUserAsync(discordEvent.Key, eventArguments.User.Id);
-                await client.SendMessageAsync(dmChannel, $"You've signed off {discordEvent.Name}.");
+                case (":arrows_counterclockwise:"):
+                    await UpdateSignupSheet(eventArguments, discordEvent, eventsSheetsService);
+                    break;
+                case (":no_entry_sign:"):
+                    await eventsSheetsService.ClearResponsesForUserAsync(discordEvent.Key, eventArguments.User.Id);
+                    await client.SendMessageAsync(dmChannel, $"You've signed off {discordEvent.Name}.");
+                    break;
+                default:
+                    try
+                    {
```

Comment:
This default case might be worth splitting out into its own function to reduce nesting? 🙂